_pages/en/target-solution-delivery-model-strategy-areas-of-concerns.md

```diff
@@ -30,353 +32,293 @@ This document will be updated throughout the elaboration of the above strategy.
-As a result, if data science is a competency that is required for a given value stream to deliver value to its users, then data science capabilities should be embedded within the team.
-As an interim step, the technologies needed for the team's competencies should be made available internally accessible so that the team is able to manage the software required to deliver the data analysis capabilities they need.
-In the case of a large monolithic piece software (which is undesirable), this could be accomplished by someone within the team having access to the required tooling in order to generate the analysis required.
-Conversely, all data should be exposed through endpoints, allowing those with access to query (or update) the information needed through automation.
-In neither case is there a team who is the gatekeeper to a given software, skillset, or dataset.
-Distributed decision making is essential to maintain agility, and highly centralized systems (without extensive investments automation) reduce agility.
-Centralizing capabilities required across projects outside of said project is undesirable.
-Management will become responsible to ensure that there is consistency regarding skill sets or technologies between teams  of teams, as their teams resources will be embedded within DevOps teams.
-In this model, teams are organized in "matrices", whereby PO's manage work vertically, and managers manage teams horizontally (based on skill sets).
-It is this horizontal management that would be responsible for best practices and 'defaults' (over standards) for a given capability (ex [though not limited to]: testing, accessibility, data science, architecture).
+DevOps teams choosing their technical stacks are still expected to provide a level of assurance in terms of maintaining the technical stack over time, including addressing IT security and funding requirements.
+Existing processes are expected to be used to provide this assurance level.
+The strategy seeks to adjust them by bringing attention to the following:
 
-- Exploring the use of Senior Advisor's model for OAS-SIS (using Squads, PO, Chief PO, Project Owner).
-The idea is that Lines of Businesses would be the POs and Project Owners.
-They would coordinate changes to IT Products downstream and prioritize them, sharing part of the accountability on IT Projects.
+<ol type="a">
+  <li>Standards should be looked at with different scope: ESDC Employee and IT Personnel.
+  As IT Personnel require the necessary tools to build/configure solutions for ESDC Employees and/or Canadians.</li>
+  <br>
+  <li>Technical stacks become obsolete much faster now (~2 years) and require revisiting standards much more often.
+  This revisit warrants more engagement with DevOps teams.</li>
+  <br>
+  <li>Use empirical evidence to make decisions, by allowing DevOps team to experiment with technical stacks, trying them out up to, and including, in production.
```

Comment:
Are we sure we want to promote teams experimenting in production? Maybe that's too big of a step to begin with?
I'm ready to be proven wrong :)

Reply:
This document explains our current thoughts, but not a decision made. I would argue that going into production grants our ability to monitor the behaviour in real world situation. This should be aligned with the "Emerging" standard category of Technical Standards Management. It doesn't mean this new tech stack is made broadly available to the whole enterprise. But to fully gain an appreciation of a particular technical stack, we should be able to bring it in production (after sufficient testing of course).

Would rather keep it as is for now. Especially to trigger such reaction by our architect peers :)

Reply:
Good points. There are distinctions between "production environments" as well. e.g., internal collaboration platforms VS external client-facing ones dealing with private information... Teams should be responsible to evaluate risks appropriately.

Agreed, let's stir up discussions :)